### PR TITLE
Patch master node with passwd for virsh console

### PR DESCRIPTION
**Pull request by celebdor**
This patchset introduces a facility for patching master nodes just like
there is for bootstrap.

### CODE REVIEW

File: utils.sh [hunk 1]
```diff
@@ -22,6 +22,54 @@ done
...
+function patch_node_ignition() {
+    local kind
+    local bootstrap
+    local wd
+    local config
+
+    kind="$1"
+    bootstrap="$2"
+    wd="$(mktemp -d)"
+    ssh_opts=("-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null")
```
Comment (etingof) [inline utils.sh]: nit: do we need parenthesis here?

File: utils.sh [hunk 2]
```diff
@@ -22,6 +22,54 @@ done
...
+
+    # Wait for the release config to be pulled
+    while
+        config=$(ssh "${ssh_opts[@]}" "core@$bootstrap" \
```
Comment (etingof) [inline utils.sh]: here and in other places where we `ssh` to the coming up host - would it make sense to tackle ssh `ConnectionAttempts` and/or `ConnectTimeout` options  in hope for more predicable timing?

File: utils.sh (hunk 1 above)
Comment (celebdor) [reply utils.sh]: I'm making it into an array. That's why I put the parenthesis there. This way I can properly quote the var when I use it.

File: utils.sh [hunk 3]
```diff
@@ -14,12 +14,63 @@ wd=$(mktemp -d)
...
+
+function patch_node_ignition() {
+    local kind
+    local bootstrap
+    local wd
+    local config
+
+    kind="$1"
+    bootstrap="$2"
+    wd="$(mktemp -d)"
```
Comment (derekhiggins) [inline utils.sh]: we appear to be leaking temp directories, but we can fix this up in another patch as there is another one to be fixed that isn't part of this patch.

**Pull request merged**